.github/workflows/close-issues.yml

```diff
@@ -0,0 +1,93 @@
+name: Close Unauthorized Issues
+
+on:
+  workflow_dispatch:
+  issues:
+    types: [opened]
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight, in case issues were missed due missing workflow runs
```

Comment:
I'd drop the cron. If something is skipped, we can do it manually. Especially when the number of issues grows, this can get ratelimited and slow, etc. I've already seen it with other actions.